Cargo.toml

```
@@ -52,6 +51,7 @@ diesel = { version = "2.0.4", features = ["postgres", "chrono", "uuid"] }
 diesel-async = { version = "0.2.2", features = ["tokio-postgres", "deadpool"] }
 diesel_migrations_async = { git = "https://github.com/aumetra/diesel_migrations_async", features = ["postgres"] }
 toml = "0.7.4"
+lettre = { version = "0.10.4", features = ["tokio1", "tokio1-native-tls", "smtp-transport", "builder"] }
```

it's better not to rely on a dynamic library like openssl, which is used when adding `tokio1-native-tls` flag

```suggestion
lettre = { version = "0.10.4", default-features = false, features = ["tokio1", "tokio1-rustls-tls", "smtp-transport", "builder"] }
```